backend/src/v2/Makefile

```diff
@@ -17,7 +17,7 @@ SHELL = /bin/bash
 
 # create an .env file to override dev vars locally
 -include .env
-DEV_IMAGE_PREFIX?=gcr.io/ml-pipeline-test/dev/kfp-
+DEV_IMAGE_PREFIX?=gcr.io/dummy/dev/kfp-
```

Comment:
Is it intended to result in an error when at `docker push`? What if someone owns a GCR repository "dummy"?
Can we require defining a variable just before make: `GCR_REPO=gcr.io/myrepo make <rule>` and add `DEV_IMAGE_PREFIX="${GCR_REPO}/dev/kfp-"` inside the Makefile?

Reply:
Updated to result explicit error before building the image if the flag is not set.

Reply:
Thanks!